colonel/wrapper/wrapper.py

```diff
@@ -22,9 +76,12 @@ def run_simulation(self,
                        duration: Optional[str] = None,
                        events_file: Optional[str] = None,
                        use_simulator_logs: bool = True,
-                       use_simulator_out: bool = True):
+                       use_simulator_out: bool = True,
+                       logged_messages: str = 'XY'):
```

Comment:
We should add a way to configure this easily. The wrapper is not an user accessible class.